README.md

```diff
@@ -443,50 +443,42 @@ specify a different token identifier:
 client.auth(nbf=nbf, exp=exp, jti=jti)
 ```
 
-## Overriding API url's
+## Overriding API Attributes
 
-By default, our API url's are hardcoded. For use cases where these url's are not accessible, best practices to override these url's are the following:
+In order to rewrite/get the value of variables used across all the nexmo classes Python uses `Call by Object Reference` that allows you to create a single client for Sms/Voice Classes. This means that if you make a change on a client instance this will be available for the Sms class.
```

Comment:
```suggestion
In order to rewrite/get the value of variables used across all the Nexmo classes Python uses `Call by Object Reference` that allows you to create a single client for Sms/Voice Classes. This means that if you make a change on a client instance this will be available for the Sms class.
```